tests/unit/flask/conftest.py

```diff
@@ -26,11 +28,12 @@ def cwd():
 @pytest.fixture(name="harness")
 def harness_fixture() -> typing.Generator[Harness, None, None]:
     """Ops testing framework harness fixture."""
+    container = "flask-app"
```

Comment:
Would it be better if this is a constant, like before the change?

Reply:
Yes, the original constant is removed, but I added a new one in the unit test module, updated, thanks!